client/__tests__/RatingsAndReviews.test.js

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
-import { render, screen, cleanup } from '@testing-library/react';
-// import { shallow } from 'enzyme';
-// import Adapter from 'enzyme-adapter-react-16';
-// * enzyme only supports up to react ^16.4.0-0, and the current dependency is 17.x
-// TODO: investigate how to replace enzyme functionality
+import { render, screen, cleanup, fireEvent } from '@testing-library/react';
 import RatingsAndReviews from '../src/components/RatingsAndReviews/RatingsAndReviews.jsx';
+import ProductDetailPage from '../src/components/ProductDetailPage.jsx';
 import { reviewsMock, reviewsMetaMock } from '../src/mockData/reviewsMock.js';
 
+var onChangeFileHandler = (event) => {
+  this.setState({
+    selectedImageFile: event.target.files[0],
+    fileLoaded: 0
+  });
+}
+
+var onClickUploadHandler = () => {
+  const data = new FormData();
+  data.append('file', this.state.selectedImageFile);
+  axios.post(`http://localhost:${port}/upload`, data, {
```

Comment:
Unsure what that empty object is for at the end. At least can we condense it and the paren onto the same line? I'm wondering if it can be removed altogether tho.

Reply:
I think that is remnants of CORS headers that I was experimenting with. It may be necessary to be more clear with headers once we tighten up the security policies on AWS.

Reply:
I'll remove and make sure it doesn't break anything.